src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/pureThermalDiffusion/pureThermalDiffusionAroundWellbore.py

```diff
@@ -0,0 +1,128 @@
+			if fsParam.get('setNames') == "{ rneg }":
+				Tin = float(fsParam.get('scale'))
+			if fsParam.get('setNames') == "{ rpos }":
+				Tout = float(fsParam.get('scale'))
+            
+	thermalConductivity = float( extractDataFromXMLList( tree.find('Constitutive/SinglePhaseConstantThermalConductivity').get('thermalConductivityComponents') )[0] )
+
+	volumetricHeatCapacity = float( tree.find('Constitutive/SolidInternalEnergy').get('volumetricHeatCapacity') )
+	
+	permeability = float( extractDataFromXMLList( tree.find('Constitutive/ConstantPermeability').get('permeabilityComponents') )[0] )
+
+	porosity = float( tree.find('Constitutive/PressurePorosity').get('defaultReferencePorosity') )
+	
+	fluidViscosity = float( tree.find('Constitutive/ThermalCompressibleSinglePhaseFluid').get('defaultViscosity') )
+
+	fluidCompressibility = float( tree.find('Constitutive/ThermalCompressibleSinglePhaseFluid').get('compressibility') )
+
+	fluidThermalExpansionCoefficient = float( tree.find('Constitutive/ThermalCompressibleSinglePhaseFluid').get('thermalExpansionCoeff') )
+
+	return [Pin, Pout, Tin, Tout, thermalConductivity, volumetricHeatCapacity, permeability, porosity, fluidViscosity, fluidCompressibility, fluidThermalExpansionCoefficient]
+
+
+def main():
+
+	xmlFilePath = "../../../../../../../inputFiles/singlePhaseFlow/thermalCompressible_2d"
+	
+	Rin, Rout = getWellboreGeometryFromXML(xmlFilePath+"_benchmark.xml")
+
+	Pin, Pout, Tin, Tout, thermalConductivity, volumetricHeatCapacity, permeability, porosity, fluidViscosity, fluidCompressibility, fluidThermalExpansionCoefficient = getLoadingFromXML(xmlFilePath+"_base.xml")
+
+	plt.figure(figsize=(10,7))
+	font = {'size'   : 16}
+	plt.rc('font', **font)
+
+	for chart_idx, idx in enumerate([1, 2, 5, 10]):
+		# Numerical results
+		data = pd.read_csv(f'data_{idx}.csv')
+		radialCoordinate = data['elementCenter:0']
+		temperature = data['temperature']
+		pressure = data['pressure']
+		diffusionTime = data['Time'][0]
+
+		# Analytical results
+		thermalDiffusionCoefficient = computeThermalDiffusionCoefficient(thermalConductivity, volumetricHeatCapacity)
+
+		T_transient = computeTransientTemperature(Tin, Rin, radialCoordinate, thermalDiffusionCoefficient, diffusionTime)
+		
+		# Analytical results of the steady state regime for comparison
+		T_steadyState = steadyState(Tin, Tout, Rin, Rout, radialCoordinate)
+
+		# Visualization
+		# Temperature
+		plt.subplot(2,2,chart_idx+1)
+		plt.plot( radialCoordinate, temperature, 'k+' , label='GEOSX' )
+		plt.plot( radialCoordinate, T_transient, 'r-' , label='Analytic' )
+		plt.plot( radialCoordinate, T_steadyState, 'b-' , label='Steady State' )
+
+		if chart_idx==1:
+			plt.legend()
+
+		if chart_idx in [2,3]:
+			plt.xlabel('Radial distance from well center')
+
+		if chart_idx in [0,2]:
+			plt.ylabel('Temperature (°C)')
+
+		plt.ylim(-10,100)
+		plt.xlim(0,1.0)
+		plt.title('t = '+str(diffusionTime)+'(s)')
+		plt.tight_layout()
+
+	plt.show()
```

Comment:
Somehow, the figure cannot be generated in the build/check of readthedocs and please double check the python script.

Reply:
The Pandas library might be missing in the online build? For instant, I added the output .png to show the validation.